azure/vm.sh

```diff
@@ -85,68 +137,83 @@ fn azure_vm_set_nics(instance, nicnames) {
 	}
 
 	nics     <= join($nicnames, ",")
-	instance <= append($instance, "--nic-names")
+	instance <= append($instance, "--nics")
 	instance <= append($instance, $nics)
 
 	return $instance
 }
 
+# azure_vm_set_storageaccount sets the storage account to "Virtual Machine".
+# Note: Only applicable when use with 'azure_vm_set_unmanageddisk'
```

Comment:
My point is that the function does not exist anywhere...why it is mentioned on the doc?

```
(klb(git addAzureManagedDisks))λ> grep -R azure_vm_set_unmanageddisk .
./azure/vm.sh:# Note: Only applicable when use with 'azure_vm_set_unmanageddisk' 
```